### ISSUE TITLE
Game of Code

### ISSUE THREAD

**Issue by petzlux** [NaelKennyGameOfCode/ParkLux#2]
Hey,  just wanted to say thanks again for your great team contribution to the Hackathon.

Would it be possible to have a functional demo of your code running on a webserver, so that we can link to it on our portal as a reuse? 

Thanks

**Comment by petzlux** [comment]
For the moment, I have linked your contribution here: https://data.public.lu/en/reuses/parklux/ 

PS: It was very easy to get your code running, great work!

**Comment by kennysong** [comment]
Yes, definitely! We have [a demo running on Azure](https://parklux.azurewebsites.net), but the free tier runs out after a month. If there is interest, perhaps we could migrate the code to run on your servers? It's essentially a static page, since all the data is on Firebase, with the exception of a cron job that needs to retrieve the parking lot data every minute or so.

**Comment by petzlux** [comment]
Okay I will put that up on the site.
